### PR TITLE
Fix conformance testdata OWNERS file.

### PR DESCRIPTION
**Pull request by cjwagner**
**What this PR does / why we need it**:
Fixes OWNERS file parse error.

**Release note**:

```release-note
none
```

/cc @mml

### CODE REVIEW

File: test/conformance/testdata/OWNERS [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # TODO(mml): Exclude parent owners once
 # https://github.com/kubernetes/test-infra/issues/5197 is implemented.
```
Comment (mml) [inline test/conformance/testdata/OWNERS]: We could remove this comment now.

**Comment by bgrant0607** [comment]
Thanks

**Pull request merged**